cookbooks/bcpc/templates/default/haproxy.cfg.erb

```diff
@@ -16,7 +16,6 @@ defaults
   maxconn  8000
   source <%="#{node[:bcpc][:management][:vip]}"%>
   mode  http
-  option httpclose
```

Comment:
I was getting a warning that it is not applicable to certain frontend(s)/backend(s) when it was global so I removed it. There there was a lot of `bcpc` related config that was removed and looks like this option was not needed anymore.